user_management/docker/requirements.txt

```diff
@@ -5,6 +5,7 @@ charset-normalizer==3.3.2
 cryptography==41.0.7
 Django==4.2.7
 django-cors-headers==4.3.1
+django-crontab==0.7.1
```

Comment:
Isn't django-crontab depracated ?
- Last release was March 2016
- Work for django 1.8
see https://pypi.org/project/django-crontab/

Reply:
django-contrab may trig some warnings, but it's still used and works with python3, as you can see.
<img width="425" alt="Screenshot 2024-02-26 at 15 04 01" src="https://github.com/tdameros/42-transcendence/assets/97832618/d669a1af-3dc5-46ac-8f26-f2859966ab69">
It works fine and it is simple.
I suggest not replace it if it's not broken so we can move on to more important tasks.
We are very late on schedule..